### ISSUE TITLE
product cards hover effect

### ISSUE THREAD

**Issue by riyarane46** [mdazfar2/Ezyshop#174]
### Description

@mdazfar2 i would like to enhance the product card  Enlarging the cards slightly when hovered would ensure that the text and images become more readable and prominent, making the website more user-friendly.

### Screenshots

![Screenshot 2024-10-05 085144](https://github.com/user-attachments/assets/6a83696a-e582-4a5f-a6c7-1a85918465c3)

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/mdazfar2/Ezyshop/blob/main/CONTRIBUTING.md)
- [X] I'm a GSSoC'24-Extd contributor
- [X] I'm a Hacktoberfest'24 contributor
- [X] I am willing to work on this issue (optional)

**Comment by mdazfar2** [comment]
@riyarane46 As of now, we do not need it. We will let you know in the future if there is a need. Please search for another issue and raise it, which I'm happy to assign to you.

- We need a store page where many local stores will be displayed, with each store categorized by name. Every category should have its own products. Let me know if you can do it, and link everything to the landing page's "Shop Now" button.

**Comment by riyarane46** [comment]
@mdazfar2 i would like to do this task . 
you can assign it to me.

**Comment by riyarane46** [comment]
there is a explore stores button in the about us section 
you want me to link the new stores page to that button right?
![Screenshot 2024-10-05 084635](https://github.com/user-attachments/assets/188ac3bc-28f9-4b53-a81d-e1be98fc913c)

**Comment by mdazfar2** [comment]
@riyarane46 yes

**Comment by riyarane46** [comment]
@mdazfar2 
can i start with the task or wait??

**Comment by mdazfar2** [comment]
@riyarane46 you can start! Can you please repeat what you should have to do?

**Comment by riyarane46** [comment]
from what i understood , 
i have to make a store page where there are different cards showing different shops 
each card should have categories of what all products they sell . eg - shop no. 1 - categories : clothes , shoes. 
whenever user clicks on a category of any shop card he will be redirected to a products page where there are products of that particular shop of the category that he chose.

@mdazfar2 please correct me if im wrong

**Comment by riyarane46** [comment]
@mdazfar2 ? please can you check my above message , it is important for me to understand my task before starting it

**Comment by mdazfar2** [comment]
Yes @riyarane46, a shop page where there are many cards, each representing a different shop name. Suppose we choose the "Fashion and Clothes" category from the category page, then we will have different shops and malls related to fashion with different names, and each shop will have different types of products or you can cloths, genes and all. Hope you understand.

**Comment by mdazfar2** [comment]
@riyarane46 feel free to reach out to me on [Discord](https://discordapp.com/users/877531143610708028) or [LinkedIn](https://linkedin.com/in/md-azfar-alam/).

**Comment by riyarane46** [comment]
okay thankyou , i will start working on it.
